### ISSUE TITLE
Does not give error of which field has errors

### ISSUE THREAD

**Issue by espen** [espen/liquid-validations#3]
```
{% name %}
{% email %}
{% street 'Adresse', 'data-required="true"' %}
{% street 'Sted', 'data-required="true"' %}
```

results in 'booking form must not have more than 1 '. Should include the field name 'street'

**Comment by espen** [comment]
Can you have a look @melashu ?

**Comment by melashu** [comment]
Hi @espen, I'm still not sure what you're asking me. Could you please give me more details?

**Comment by espen** [comment]
When the Liquid input is as above it will not give proper error message. It gives error "booking form must not have more than 1". but it should include the error field: "booking form must not have more than 1 street". It might be related to using an array of values so the error does not know which field caused the error.

I am using this:
`validates_liquid_tag :booking_form, tag: ['date_of_birth', 'national_id_no', 'street', 'postal_code', 'city', 'country_code', 'count', 'note'], max: 1, presence: false, :allow_blank => true`

**Comment by melashu** [comment]
Hi @espen, this happens because `{% street 'Sted', 'data-required=" true"' %}` is treated as an attribute that is not present in the tag list instead of`street`.  For example if you make it like `"{% name %} {% email %} {% street %} {% street %}"` you can get a correct error message. I think we need to write another logic that can extract such like pattern from an array of tags!

**Comment by melashu** [comment]
we didn't consider this in the previous senario

**Comment by espen** [comment]
I thought that was already working. Is it just a problem when using an array, not a string? I thought there were tests for this pattern

**Comment by melashu** [comment]
Yes this is when there is an array, for string it works fine and there is a test for that too

**Comment by melashu** [comment]
look this 
```ruby 
  it "must be valid when using more complex tag" do
        @mixin.content = "{% josh_is_awesome foobar, data-required='true' %}"
        @mixin.valid?
        @mixin.errors.full_messages.any? { |e| e == "You must supply {% josh_is_awesome %} in your content" }.must_equal false
      end


```

**Comment by espen** [comment]
Ok, can you make a failing test for using with array? Then see if you can fix it.

**Comment by melashu** [comment]
Sure, I will check it
